subprojects/docs/src/samples/userguide/artifacts/defineRepository/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,305 @@
+// tag::maven-ivy-repository-no-auth[]
+repositories {
+    maven {
+        setUrl("http://repo.mycompany.com/maven2")
```

Comment:
Use the overload `maven(url=...)`

Reply:
I didn't noticed this overload existed. But I also tried to stay as close as the groovy script as possible. Let's see what the team members prefer here.